huaweicloud/services/acceptance/ucs/resource_huaweicloud_ucs_policy_test.go

```diff
@@ -100,6 +101,7 @@ func TestAccPolicy_basic(t *testing.T) {
 				),
 			},
 			{
+				Config:            testPolicy_basic_import(name),
```

Comment:
why we need it?

Reply:
because updating the method will check the number of outputs in the Terraform configuration, resulting in errors in the imported method.